specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
+      },
+      "SoftwareUpdateProperties": {
+        "description": "Properties of the software update object.",
+        "properties": {
+          "updateState": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Needed / Installed"
+          },
+          "availableOnComputerSince": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Point in time when this update was first seen on this computer"
+          },
+          "approved": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Approval status"
+          },
+          "packageRepository": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Package repository"
+          },
+          "supercededUpdateIds": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Superseded Update Ids"
+          },
+          "isSupercededUpdate": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Is a superceded update"
+          },
+          "deploymentAction": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Deployment action"
+          },
+          "kbid": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Knowledge Base ID of a Windows Update"
+          },
+          "kbRevisionNumber": {
+            "type": "string",
+            "readOnly": true,
+            "description": "KB revision number"
+          },
+          "publishedDate": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Reported published date for this update"
+          },
+          "productName": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Product name"
+          },
+          "productVersion": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Product version"
+          },
+          "productArch": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Product architecture"
+          },
+          "title": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Update title"
+          },
+          "classification": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Normalized update classification"
+          },
+          "packageClassification": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Linux package classification"
+          },
+          "packageSeverity": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Linux package severity"
+          },
+          "msrcSeverity": {
```

Comment:
what are the allowed values? string enum?

Reply:
Same as packageSeverity. The list is not constrained and may change depending on OS, so keeping it free form string